packages/portal-proto/src/components/Table/ColumnOrdering.tsx

```diff
@@ -81,16 +86,18 @@ function ColumnOrdering<TData>({
   };
 
   return (
-    <div ref={ref}>
+    <Menu
+      position="bottom-end"
+      middlewares={{ shift: false, flip: false }}
+      onChange={setShowColumnMenu}
+      offset={0}
+    >
       <Tooltip label="Customize Columns" disabled={showColumnMenu}>
-        <span>
+        <Menu.Target>
           <ActionIcon
             variant="outline"
             size="lg"
-            onClick={() => setShowColumnMenu((o) => !o)}
-            aria-label={`${
-              showColumnMenu ? "close" : "show"
-            } column change menu button`}
+            aria-label="Customize Columns"
```

Comment:
Nitpick: I feel like the prior was more informative. I am fine with it though. @ocshawn @wteouchicago

Reply:
when using the menu component it automatically adds a aria-expanded property this makes the inclusion of any text that describe the state unneeded as its handled by that